### ISSUE TITLE
get listed among gentoo overlays

### ISSUE THREAD

**Issue by pavlix** [fosero/flatpak-overlay#6]
I couldn't see flatpak at http://gpo.zugaina.org/Search?search=flatpak, which means it hasn't been added to the Gentoo overaly list and cannot be directly installed with `layman -a flatpak` or similar. 

See also:

  * https://overlays.gentoo.org/
  * https://wiki.gentoo.org/wiki/Project:Overlays/Overlays_guide

**Comment by fosero** [comment]
I'm sort of lazy :) and I really do not want any more than receive bug reports about it through github.

**Comment by ghost** [comment]
It's really not that bad, even I have an overlay thats listed: https://gpo.zugaina.org/Overlays/netcrave

IIRC you open a ticket on bugs.gentoo.org and you just need an account (register) I think this is the ticket template: https://bugs.gentoo.org/enter_bug.cgi?product=Gentoo+Infrastructure&component=Gentoo+Overlays

**Comment by ghost** [comment]
Without proper integration into repository system, this isn’t an overlay.

1. Repositories in Gentoo – including those in layman – are synchronised with `emaint-sync`. And people synchronise them every day. This repository, however, requires to synchronise it manually.

2. Ebuilds in this repository can’t pass quality checks and EAPI compatibility, as technically it’s not in the repository system. In fact that means that this repo might not be at all, as the ebuilds aren’t guaranteed to work.

**Comment by fosero** [comment]
@deterenkelt Really, you're gonna talk about semantics here? Some way to keep the motivation up. You don't like it? Don't use it.

1. You can have this repo sync automatically just fine.
2. I'm not really interested in providing the perfectly correct ebuild. I've been a Gentoo dev, I've done my share of ebuilds. No I don't keep up with the latest fads in ebuildland, but these ebuilds are correct and fine. If you find a problem, report them and (if possible) provide a fix and it will only get better.

If someone wants to do an overlay for Gentoo and add it to the 'official' overlays, feel free. I just did this one for my own benefit and shared it in the hope other people found it useful as well. I'm not interested in providing anything beyond this.

**Comment by ghost** [comment]
Ok, just rename the repo to “scrapfolder for a gentoo overlay” to not confuse other people.
